.changes/README.md

```diff
@@ -0,0 +1,27 @@
+# CHANGELOG Automation
+
+We use [changie](https://changie.dev/) to automate `CHANGELOG` generation.  For installation and format/command specifics, see the documentation.
+
+### Workflow
+
+#### Daily workflow
+Each code change gets a changelog file by running `changie new` and following the prompts.  This ensures correct file format and file name.
```

Comment:
Just to clarify here, do I run `changie new` on my local dev branch after I have created a PR? Then push that new changie file up to my PR? I can add more commits to that PR and that doesn't matter, right?

Reply:
That is correct.

`changie new` simply creates a yaml file in `.changes/unreleased`.  You can create as many commits as you want and that shouldn't matter.  You can also directly edit the generated yaml file as well if you needed to update your change description, etc.

I'll try to reword this to be a bit more clear.